src/components/PlioQuestion.vue

```diff
@@ -212,37 +214,35 @@ export default {
       document.querySelector("body").classList.remove("overflow-hidden");
     },
 
-    selectOption(option_index) {
+    selectOption(optionIndex) {
       this.$emit("update-journey", "option-selected", {
         question: Number(this.plioQuestion.id),
-        option: option_index,
+        option: optionIndex,
       });
     },
 
     // Opens the question window
     openModal() {
       // Show highlighted options if coming back to an answered question
       // Wait 200 ms because it takes some time to find the DOM elements
-      if (this.plioQuestion.state == "answered") {
+      if (this.isAnAnsweredQuestion) {
         requestAnimationFrame(() => {
+          this.isAnswerSubmitted = true;
+
           // highlight wrong/right depending on what the user answered in previous session
-          document.getElementById("options_container").classList.add("options-block");
+          // document.getElementById("options-container").classList.add("options-block");
           var selectedOption = document.getElementById(
-            `option_input_${this.userAnswerIndex}`
+            `option_input_${this.plioQuestion.userAnswer}`
           );
           selectedOption.checked = true;
-          this.selectedOption = this.plioQuestion.user_answer;
-          this.checkAnswer();
+          this.selectedOption = this.plioQuestion.item.details.options[
+            this.plioQuestion.userAnswer
```

Comment:
Can we mention somwhere that `userAnswer` is actually an index and not the text?